HotelBooking/Controllers/reservation_controller.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from tracemalloc import start
+from typing import List
+from HotelBooking.Controllers.controller import Controller
+from HotelBooking.Controllers.bill_controller import BillController
+from HotelBooking.Models import reservation
+from HotelBooking.Models.bill import Bill
+from HotelBooking.Models.reservation import Reservation
+from HotelBooking.Models.room import Room
+from dateutil import parser
+
+from HotelBooking.Models.roomType import RoomType
+
+# Cost per hour of stay
+ROOM_PRICING = {
```

Comment:
Did you not add the room pricings to the database? If so, use the pricing in the database.